contrib/scripts/env-setup.py

```diff
@@ -25,7 +25,7 @@
 VERSION_REGEX = re.compile(r'^(\d+\.)+\d+$')
 
 def is_gpu_runtime():
-  return int(os.environ.get('COLAB_GPU', 0)) == 1
+  return os.environ.get('COLAB_GPU', 0) == '1'
```

Comment:
if you mean using getenv_as in line 28 of contrib/scripts/env-setup.py ( after declaring the same function In this module) like so :

return getenv_as(name = "COLAB_GPU", type = int, defval = 0) == 1 

I still get the same error [  ValueError: invalid literal for int() with base 10: '' ] because :

- (in the getenv_as() function declaration) os.environ.get(name, None) returns an empty string that is stored in env.
- type(env) attempts to typecast env as an int (according to the above function call)  which throws the error mentioned above

Reply:
I see, this makes sense for now. 

I'll follow up to add empty string handling to `getenv_as`